src/models/methods/tx.ts

```diff
@@ -0,0 +1,19 @@
+import { TransactionMetadata } from "../common/transaction";
+import { BaseRequest, BaseResponse } from "./baseMethod";
+
+export interface TxRequest extends BaseRequest {
+  command: "tx"
+  transaction: string
+  binary?: boolean
+  min_ledger?: number
+  max_ledger?: number
+}
+
+export interface TxResponse extends BaseResponse {
+  result: {
+    hash: string
+    ledger_index: number
+    meta: TransactionMetadata | string
+    validated?: boolean
```

Comment:
Is there a field for `transaction` (or something similar) here? Is that what the comment on the next line is referring to?

Reply:
Yeah, all the `transaction` fields are a part of `result`, so the final thing will basically be `{...} & Transaction`